packages/bindings/src/serialport.cpp

```diff
@@ -38,7 +38,8 @@ NAN_METHOD(Open) {
     Nan::ThrowTypeError("First argument must be a string");
     return;
   }
-  v8::String::Utf8Value path(Nan::To<v8::String>(info[0]).ToLocalChecked());
```

Comment:
Does it work if
```cpp
  v8::String::Utf8Value path(Nan::To<v8::String>(info[0]).ToLocalChecked());
```
is replaced with
```cpp
  Nan::Utf8String path(info[0]);
```
rather than replacing it with
```cpp
  v8::Isolate* isolate = info.GetIsolate();
  v8::String::Utf8Value path(isolate, Nan::To<v8::String>(info[0]).ToLocalChecked());
```

Reply:
Yes, that appears to work. Thanks!